<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripManager</class>
 <widget class="QWidget" name="TripManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="TicketList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFile">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditTicket">
       <property name="text">
        <string>编辑车票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveTicket">
       <property name="text">
        <string>删除车票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newTicket">
       <property name="text">
        <string>添加车票</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
